src/Sylius/Bundle/PaymentBundle/spec/Form/Type/PaymentMethodTypeSpec.php

```diff
@@ -40,24 +40,10 @@ function it_is_a_form_type()
         $this->shouldImplement(FormTypeInterface::class);
     }
 
-    function it_builds_form_with_proper_fields(
-        FormBuilder $builder
-    ) {
-        $builder
-            ->add('translations', 'sylius_translations', Argument::any())
-            ->shouldBeCalled()
-            ->willReturn($builder);
-
-        $builder
-            ->add('enabled', 'checkbox', Argument::any())
-            ->willReturn($builder)
-        ;
-
-        $builder
-            ->add('gateway', 'sylius_payment_gateway_choice', Argument::any())
-            ->willReturn($builder)
-        ;
-
+    function it_adds_add_code_event_subscriber(FormBuilder $builder) 
+    {
+        $builder->add(Argument::any(), Argument::cetera())->willReturn($builder);
```

Comment:
It does not check anything, deleting the whole spec would be the better option 🗡